### ISSUE TITLE
mw:194107

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-orig#74]
date:	11/21/2019 02:00:22
dict:	mw
Lnum:	194107
hw:	viklRp
old:	Second attempt: I made a mistake on the first: let me try again: The citation searched for was "viklRp", produces headword "vikḷp" and the first (H1) entry is "vi-√ kḷp". The search for "vikLp", on the other hand, returns no match.
new:	After looking at MW's "Dictionary of Nagari order" facing page 1: the headword IS incorrect AND there should be no match to it when the citation entered is "vikLp". Furthermore, MW pg 955,1 shows indeed devanagari for "viklṛp" and the MW's first entry is actually "vi-√ klṛp", and that root √ klṛp is indeed found on MW pg 308 column 2. So then, only citation for "viklRp" should match. The medial form of the letter under the "k" stands for "lṛ" (or in his terminology "lṛi" as in revelry (revelṛi).
comm:	Typo or mismatch from MW's designation of "lṛi" instead of every other grammarians designation of "ḷi" for the same nagari letter after "ṛī".
email:	jimgknga@zoho.com

**Comment by funderburkjim** [comment]
This is the submitter's rewrite of #73.  See comment in #73.
